# This message is for storing a segment of interest (i.e. segmentation results) in a generic format
#  Raymond Kirk (Tunstill) Copyright (c) 2020
#  Email: ray.tunstill@gmail.com

float64[] x  # Array of [x1 x2 ... xn] in image co-ordinates
float64[] y  # Array of [y1 y2 ... yn] in image co-ordinates